### ISSUE TITLE
Should exhibition all operations 

### ISSUE THREAD

**Issue by erdun** [oam-dev/spec#371]
Oam create the ecosystem of operation, should provide a place to show all operations, and sppourt other developers upload new operation. Help the end-users use oam more simply

**Comment by resouer** [comment]
@erdun Yes! I think you are talking about a Traits Registry, correct? /cc @wonderflow @ryanzhang-oss

**Comment by wonderflow** [comment]
Yes, we are designing a micro-app engine core for k8s based on OAM.  Traits Registry will be included in it.
